tests/device_tests/test_msg_change_wipe_code_t2.py

```diff
@@ -143,6 +143,7 @@ def test_set_remove_wipe_code(client):
     assert client.features.wipe_code_protection is False
 
 
+@pytest.mark.setup_client()
```

Comment:
iirc you never need `setup_client` without arguments? the defaults will be used anyway